components/log-viewer-webui/client/eslint.config.mjs

```diff
@@ -28,6 +28,7 @@ const EslintConfig = [
     ...ReactConfigArray,
     {
         rules: {
+            "dot-notation": "off",
```

Comment:
Let's turn it off for TS only. like instead, we want to add this object to the end of the `EslintConfig` array:
```
{
     'files: [
         "**/*.ts",
         "**/*.tsx",
     ],
     rules: {
         "dot-notation": "off",
         "@typescript-eslint/dot-notation": "error",
     },
},
```